src/main.rs

```diff
@@ -318,6 +324,18 @@ fn main() {
         );
     }
 
+    if let Some(gcno) = opt.gcno {
```

Comment:
Should we make this option conflict with others, given that we return early when it is set?

Reply:
How do you want to do that ?